indico/modules/events/operations.py

```diff
@@ -42,3 +42,11 @@ def delete_reference_type(reference_type):
     db.session.delete(reference_type)
     db.session.flush()
     logger.info('Reference type "%s" deleted by %s', reference_type, session.user)
+
+
+def create_event_references(event, reference_data):
+    references = [EventReference(reference_type_id=int(r['type']), value=r['id']) for r in reference_data]
+    event.references = references
+    db.session.flush()
+    for reference in references:
+        logger.info('Reference "%s" created by %s', reference, session.user)
```

Comment:
That's pretty spammy since it doesn't only log changed ones. Anyway, it's just the logfile so it's not really an issue.